### PR TITLE
Added comment field to na_ontap_volume module

### PR DESCRIPTION
**Pull request by joshedmonds**
##### SUMMARY
Add support for the comment field to na_ontap_volume.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
- na_ontap_volume.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by lonico — APPROVED** [review]
shipit

thank you for fixing the typo in volumes-qos-attributes

File: lib/ansible/modules/storage/netapp/na_ontap_volume.py [hunk 1]
```diff
@@ -684,6 +697,8 @@ def create_volume_options(self):
             options['export-policy'] = self.parameters['policy']
         if self.parameters.get('junction_path'):
             options['junction-path'] = self.parameters['junction_path']
+        if self.parameters.get('comment'):
+            options['comment'] = self.parameters['comment']
```
Comment (lonico) [inline lib/ansible/modules/storage/netapp/na_ontap_volume.py]: this field shows as volume-comment in 9.6 ZAPI documentation.   Is this version dependent?

Confirmed that create is using volume-comment but modify is using comment :(

**Review by lonico — CHANGES_REQUESTED** [review]
comment should be volume-comment for create

File: lib/ansible/modules/storage/netapp/na_ontap_volume.py (hunk 1 above)
Comment (joshedmonds) [reply lib/ansible/modules/storage/netapp/na_ontap_volume.py]: Sorry, I missed that ZAPI discrepancy between the create, modify and get calls. Now fixed.

**Review by lonico — APPROVED** [review]
shipit

**Comment by lonico** [comment]
ship

**Review by lonico — APPROVED** [review]
shipit

**Review by thedoubl3j — APPROVED** [review]
lgtm. feature added and test coverage added for it. shipit

**Pull request merged**